src/main/java/seedu/clinic/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.clinic.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.clinic.logic.commands.exceptions.CommandException;
+import seedu.clinic.model.Model;
+
+public class RedoCommand extends Command {
+    public static final String COMMAND_WORD = "redo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Redo the previously undone command"
+            + " if the undone was successfully executed\n\n"
```

Comment:
I think it should be "if undo was successfully executed" here :)

Reply:
Great suggestion, will fix it!